.changeset/tasty-buses-jog.md

```diff
@@ -0,0 +1,114 @@
+---
+'@keystone-next/keystone': major
+---
+
+The names of two-sided many-many relationships now only include the list key and field key for one side of the relationship and one-sided many relationships(which are many-many) no longer have `_many` at the end of them for consistency with two-sided relationships. This allows having many-many relationships with long list and field keys without hitting Postgres's 63 character limit.
```

Comment:
```suggestion
The names of two-sided many-many and one-to-one relationships has have been shortened to the left hand side only.
This reduces the probability that you will exceed [POSTGRESQL's 63 character limit for identifiers](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) with typical usage.

This is a breaking change. 

```

Reply:
> one-to-one relationships has have been shortened to the left hand side only

This isn't correct and you've removed the mention of one-sided many relationships.

Reply:
Fixed to read `one-to-many`

Reply:
It's not about one to many, it's about one sided many relationships(which are many to many) - https://keystonejs.com/docs/guides/relationships